_projects/jetbrains-ide-plugin.md

```diff
@@ -2,7 +2,8 @@
 collaborating_projects:
   - coala
   - jetbrains
-desc: "Offer a general mechanism of highlighting code analysis results from external tools in jetbrain IDE, the primary goal is to support coala. "
+desc: "Offer a general mechanism of highlighting code analysis results from external tools in jetbrain IDE, the primary
```

Comment:
break this line at 80cpl, and align indented text on next line under first letter of this line.